### PR TITLE
[SD-1322] Added content expiry field for the stage settings

### PR DESCRIPTION
**Pull request by akintolga**
This pull request also tackles the task:
[SD-1325] Add expiry time-span field per desk

Rebased on the changes regarding the task status - stage connection.

### CODE REVIEW

File: app/scripts/superdesk-desks/directives.js [hunk 1]
```diff
@@ -80,16 +80,57 @@ define([
         return {
 
             link: function(scope, elem, attrs) {
-                scope.origEditName = null;
+                scope.editingStage = null;
+
+                scope.createNewStage = function(step, desk) {
```
Comment (petrjasek) [inline app/scripts/superdesk-desks/directives.js]: seems like those arguments are not used

File: app/scripts/superdesk-desks/views/settings.html [hunk 2]
```diff
@@ -69,24 +73,53 @@ <h3 ng-show="desk.edit._id" translate>Edit Desk</h3>
...
+								<div sd-deskedit-basic>
+									<form name="generalStep">
+										<div class="content">
+											<fieldset>
+												<div class="field">
+													<label translate>Stage name</label>
+													<input type="text" class="fullwidth-input" ng-model="editingStage.name" required> 
+												</div>
+												<div class="field">
+													<label translate>Stage description {{test}}</label>
```
Comment (petrjasek) [inline app/scripts/superdesk-desks/views/settings.html]: what are you testing? :smile:

Comment (akintolga) [inline app/scripts/superdesk-desks/views/settings.html]: It looks like I was testing copy-paste !! :flushed:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**